packages/tasks/src/model-libraries.ts

```diff
@@ -661,6 +661,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: true,
 		countDownloads: `path:"pytorch_model.bin" OR path:"model.safetensors"`,
 	},
+	imstoucan: {
```

Comment:
There is currently no models tagged as `imstoucan` (see [here](https://huggingface.co/models?other=imstoucan)). Could you add `library_name: imstoucan` to the modelcard metadata of https://huggingface.co/Flux9665/ToucanTTS to reference it properly?

Reply:
Thanks for the hint, I added the library_name to the modelcard.